olive/passes/onnx/genai_model_exporter.py

```diff
@@ -93,7 +94,8 @@ def _run_for_config(
         if not cache_dir:
             cache_dir = os.environ.get("TRANSFORMERS_CACHE", None)
         if not cache_dir:
-            cache_dir = output_model_filepath.parent / "genai_cache_dir"
+            # please do not clean up the cache dir as it may contain model's data
+            cache_dir = str(Path(tempfile.gettempdir()) / "hf_cache")
```

Comment:
how could user know the cache folder after the pass done since it is a temp dir?

Reply:
or need we use the output_model_path as cache_dir?

Reply:
![image](https://github.com/microsoft/Olive/assets/13343117/1c0d70ad-51c7-48ec-ab9e-05a4f3701fbf)
I just tried to run model build scripts and find that the `cache_dir` is used to:
1. store the downloaded huggingface model snapshot which plays the same role with huggingface's cache folder.
2. besides, model builder will use this cache dir to generate temporal onnx model weights which will be removed after the conversion.

So after the conversion, the cache folder only has the context of #1. User do not need to know where is the cache dir.